yeonjae/programmers-1/문자열내p와y의개수.js

```diff
@@ -0,0 +1,29 @@
+function solution(s){
+  const sameLevelStr = s.toLowerCase();
+  
+  let lengthP = 0;
+  let lengthY = 0;
+      
+  for(let i=0; i<sameLevelStr.length; i++) {
+    if(sameLevelStr[i] === "p") lengthP++
+    else if(sameLevelStr[i] === "y") lengthY++
+    else true;
+  }
+
+  if(lengthP === lengthY) return true;
+  else return false;
+}
```

Comment:
```suggestion
function solution(s){
  const sameLevelStr = s.toLowerCase();
  
  let pCount = 0; 
  let yCount = 0;
      
  for (let i = 0; i < sameLevelStr.length; i++) {
    if (sameLevelStr[i] === "p") lengthP++
    else if (sameLevelStr[i] === "y") lengthY++
    else true; // 이 부분 없어도 될 것 같아유!!  
  }
  
  return lengthP === lengthY;
}
```
- length라는 변수보다는 반복문을 돌리면서 카운트하기때문에 count가 좀 더 직관적인 것 같은데 어떻게 생각하세용? 😮
- 하연님께도 비슷한 의견을 드렸었던에 일반 for문보다 `for...of`문이 좀 더 깔끔하다는 느낌이 듭니다 (하연님 리뷰 참고)
- 마지막 리턴도 곧 바로 `lengthP === lengthY`의 비교값을 리턴하면 어떨까요? if ... else 문을 사용하는 것보다 훨씬 깔끔해보입니다 👍
- 그리고 if문 뒤로 띄어쓰기, 연산자 좌우로 띄어쓰기 등 신경써주세요 아니면 vscode 자체에 prettierr 설정해주면 좋을 것 같아요